mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -13,8 +13,6 @@
 
 #include "tier0/memdbgon.h"
 
-#define SAVELOC_FILE_NAME "savedlocs.txt"
```

Comment:
It's not important for the failure message to have reference to this file name. You can instead keep the define in here and just in the dev logs do "the savelocs file"

Reply:
This has now been moved back to the cpp file.